NASA-AMMOS/aerie-ui
# Add feature for defining model specifications

resolves #1212 

To test:
1. Create constraint(s)
1. Create scheduling condition(s)
1. Create scheduling goal(s) 
1. Create a model
1. Click on the newly created model in the models table
1. Click on the new "Edit details and associations" button on the left, under the model's metadata
1. Verify that you are taken to the new edit model page
1. Modify the model metadata (i.e. name, description, etc)
1. Click on the "Constraints" tab
1. Click the "Library" tab
1. Toggle on the checkboxes for all the constraints that you want to add
1. Click the "Model" tab
1. Verify that all the constraints that you toggled on are present
1. Choose specific versions of the constraints that you prefer
1. Click on the "Conditions" tab and repeat steps 10 - 14, but for scheduling conditions
1. Click on the "Goals" tab and repeat steps 10 - 14, but for scheduling goals
1. For goals, also update the priority if desired
1. Click the "Save" button at the top right
1. Verify that all changes are persisted after a refresh
